### ISSUE TITLE
А можно как-то всплывающую подсказку при наведении на картинку убрать?

### ISSUE THREAD

**Issue by user55** [SthephanShinkufag/Dollchan-Extension-Tools#1258]
Если навести на превью картинки и ничего не нажимать, появится "[Click] открыть по центру, [Ctrl+Click] в посте". А у меня установлен imagus, он автоматически картинки увеличивает при наведении. И получается так, что на картинке или видео эта подсказка поверх маячит.

**Comment by SthephanShinkufag** [comment]
Даже не знаю. Опцию ради такой мелочи делать не хочется, а если выпилить подсказку &mdash; значит пользователи не будут знать про `Ctrl+Click`.

**Comment by SthephanShinkufag** [comment]
Отключите в куклоскрипте разворот картинок (`"Картинки"` > `"[Откл.] Раскрывать картинки по клику"`), и подсказки не будет.

**Comment by user55** [comment]
Отключить-то отключу, но я этой фишкой тоже иногда пользуюсь. Может просто в выпадающем меню опции переименовать ("Картинки" > "[[Click] по центру, [Ctrl+Click] в посте] Раскрывать картинки по клику"), и, соответсвенно, наоборот  ("Картинки" > "[[Click] в посте, [Ctrl+Click] по центру] Раскрывать картинки по клику")? Так и пользователи будут замечать эту опцию при настройке куклоскрипта. А так может вообще не обращают внимания, лично я вообще не обращал до установки имагуса.

**Comment by SthephanShinkufag** [comment]
Никто не смотрит в настройки, к сожалению. так хоть как-то заметят.

**Comment by SthephanShinkufag** [comment]
Зачем вам куклоскриптовая раскрывалка, если вы пользуетесь сторонней?

**Comment by user55** [comment]
Иногда я раскрываю посты по центру и прокручиваю все стрелочками, Кстати, я еще люблю прокручивать картинки колесом мышки в просмотрищках - очень удобно. Может подумаете над запиливанием такого функционала? Поскольку картинка открывается на весь экран, я думаю особо нет нужды зумить ее или уменьшать тем более. Или можно сделать это опционально - зум или прокрутка. По поводу, что никто не смотрит, это ведь все-таки скрипт для "продвинутых" пользователей, так? И если человек устанавливает, но не удосуживается залезть в настройки, это же его проблема?

**Comment by SthephanShinkufag** [comment]
> Поскольку картинка открывается на весь экран, я думаю особо нет нужды зумить ее или уменьшать тем более.

А для чего, по-вашему, в скрипте реализован зум картинок? Я им постоянно пользуюсь, например, чтобы рассмотреть детали.

**Comment by SthephanShinkufag** [comment]
>  нет нужды уменьшать тем более

Я уменьшаю картинку, если она перекрывает какой-нибудь пост, например. То, что вы этим не пользуетесь, не значит, что никто не пользуется.

**Comment by SthephanShinkufag** [comment]
Сегодня какой-то день странных реквестов.

**Comment by user55** [comment]
Вообще-то вспомнил, что я тоже пользуюсь зумом иногда. Но над идеей подумайте все-таки, может как опцию на выбор запилите.

**Comment by ImoutoChan** [comment]
Можно прокручивать с зажитым shift ом, а без него зумить. Видел в каких-то аппах такое.

**Comment by user55** [comment]
Да. Или сделать опцию, что зажатие шифта активирует - прокрутку или зум.
